cg/cli/store/store.py

```diff
@@ -1,46 +1,153 @@
+@click.pass_context
+def store_ticket(context: click.Context, ticket: str, dry_run: bool) -> None:
+    """Include links to decompressed FASTQ files belonging to a ticket in Housekeeper."""
+    status_db: Store = context.obj.status_db
+    samples: list[Sample] = status_db.get_samples_from_ticket(ticket=ticket)
+    stored_individuals: int = invoke_store_samples(
+        context=context, dry_run=dry_run, sample_ids=[sample.internal_id for sample in samples]
+    )
+    LOG.info(f"Stored fastq files for {stored_individuals} samples")
+
+
+@click.command("demultiplexed-flow-cell")
+@click.argument("flow-cell-id", type=str)
+@DRY_RUN
+@click.pass_obj
+def store_demultiplexed_flow_cell(context: click.Context, flow_cell_id: str, dry_run: bool) -> None:
+    """Include all files in a flow cell bundle and its corresponding sequencing files. Updates SPRING metadata file"""
+    compress_api: CompressAPI = context.meta_apis["compress_api"]
+    status_db: Store = context.status_db
+    update_compress_api(compress_api, dry_run=dry_run)
+
+    samples: list[Sample] = status_db.get_samples_from_flow_cell(flow_cell_id=flow_cell_id)
+    bundle_names: list[str] = (
+        [sample.internal_id for sample in samples if sample.internal_id] + [flow_cell_id]
+        if samples
+        else [flow_cell_id]
+    )
+    for bundle_name in bundle_names:
+        compress_api.hk_api.include_files_to_latest_version(bundle_name=bundle_name)
+
+    if not samples:
+        LOG.info(f"No samples found for flow cell {flow_cell_id} in the database")
+        return
+    for sample in samples:
+        spring_metadata_files: list[File] = compress_api.hk_api.get_files_from_latest_version(
+            bundle_name=sample.internal_id, tags=[SequencingFileTag.SPRING_METADATA]
+        )
+        if spring_metadata_files:
+            for spring_metadata_file in spring_metadata_files:
+                LOG.info(
+                    f"Updating file paths in SPRING metadata file {spring_metadata_file.full_path}: for sample: {sample.internal_id}"
+                )
+                update_metadata_paths(
+                    spring_metadata_path=spring_metadata_file.full_path,
+                    new_parent_path=Path(spring_metadata_file.full_path).parent,
+                )
+
+
+@click.command("qc-metrics")
+@click.argument("case-id", type=str)
+@DRY_RUN
+@click.pass_obj
+def store_qc_metrics(config: CGConfig, case_id: str, dry_run: bool = False) -> None:
+    """Fetch the QC metrics for a case from Janus and Store them in Arnold."""
+    metrics_api = CollectQCMetricsAPI(
+        hk_api=config.housekeeper_api,
+        status_db=config.status_db,
+        janus_api=config.janus_api,
+        arnold_api=config.arnold_api,
+    )
+    try:
+        metrics_api.create_case(case_id=case_id, dry_run=dry_run)
+    except (JanusServerError, JanusClientError, ArnoldClientError, ArnoldServerError) as error:
+        LOG.info(f"Could not store qc metrics in arnold. Reason {error}")
```

Comment:
I know our design pattern has been to create modular CLI commands and then invoke a bunch of them within other CLI-commands. I think this is a bad pattern and would like to move away from it. I think this line can be added directly to `cg.cli.workflow.commands.store`.
`metrics_api.create_case(case_id=case_id, dry_run=dry_run)`

Reply:
Hmm, but maybe prod would like to be able to add stuff manually? In that case I like a CLI-command but would like to avoid `invoke` within the store command

Reply:
The intent for this CLI is indeed manual intervention in the process if required as well as debugging purposes. When needed. 

I agree with moving away from the usage of `invoke`. I will make a CG-EP regarding this.

Reply:
CG-EP can be found here https://github.com/Clinical-Genomics/atlas/pull/2423